vips/vip-245.ts

```diff
@@ -0,0 +1,235 @@
+    oldBorrowSpeed: emissionPerBlock("4.68").div(2).toString(),
+  },
+  {
+    name: "vXRP",
+    vToken: "0xB248a295732e0225acd3337607cc01068e3b9c10",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("2.86").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("2.86").div(2).toString(),
+  },
+  {
+    name: "vLINK",
+    vToken: "0x650b940a1033B8A1b1873f78730FcFC73ec11f1f",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("2.86").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("2.86").div(2).toString(),
+  },
+  {
+    name: "vDOT",
+    vToken: "0x1610bc33319e9398de5f57B33a5b184c806aD217",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("4.58").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("4.58").div(2).toString(),
+  },
+  {
+    name: "vMATIC",
+    vToken: "0x5c9476FcD6a4F9a3654139721c949c2233bBbBc8",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("3.12").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("3.12").div(2).toString(),
+  },
+  {
+    name: "vLTC",
+    vToken: "0x57A5297F2cB2c0AaC9D554660acd6D385Ab50c6B",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("2.86").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("2.86").div(2).toString(),
+  },
+  {
+    name: "vFIL",
+    vToken: "0xf91d58b5aE142DAcC749f58A49FCBac340Cb0343",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("1.56").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("1.56").div(2).toString(),
+  },
+  {
+    name: "vDAI",
+    vToken: "0x334b3eCB4DCa3593BCCC3c7EBD1A1C1d1780FBF1",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("4.86").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("4.86").div(2).toString(),
+  },
+  {
+    name: "vDOGE",
+    vToken: "0xec3422Ef92B2fb59e84c8B02Ba73F1fE84Ed8D71",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("1.56").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("1.56").div(2).toString(),
+  },
+  {
+    name: "vBCH",
+    vToken: "0x5F0388EBc2B94FA8E123F404b79cCF5f40b29176",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("1.56").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("1.56").div(2).toString(),
+  },
+  {
+    name: "vAAVE",
+    vToken: "0x26DA28954763B92139ED49283625ceCAf52C6f94",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("1.56").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("1.56").div(2).toString(),
+  },
+  {
+    name: "vTUSD",
+    vToken: "0xBf762cd5991cA1DCdDaC9ae5C638F5B5Dc3Bee6E",
+    supplySpeed: "0",
+    borrowSpeed: "0",
+    oldSupplySpeed: emissionPerBlock("3.12").div(2).toString(),
+    oldBorrowSpeed: emissionPerBlock("3.12").div(2).toString(),
+  },
```

Comment:
Not needed. It was already set to zero in the [VIP-240](https://app.venus.io/#/governance/proposal/240?chainId=56)